README.md

```diff
@@ -50,7 +50,7 @@ As the tool is still somewhat rough, its error handling is not very good at the
 When making changes, be sure to use [swift-format](https://github.com/apple/swift-format) on the source:
 
 ```console
-$ swift-format -i *.swift
```

Comment:
I largely consider the makefile to be convenience rather than the advertised ways to do things